### ISSUE TITLE
running under scheduled task

### ISSUE THREAD

**Issue by mcconnellt150** [christianspecht/scm-backup#30]
Hi All,

when I try to run this as a scheduled task I get errors like the below, is there anyway I can correct this?

Info SCM Backup 1.2.0.870cd29
Info http://scm-backup.org/
Error Could not find file 'C:\Windows\system32\NLog.config'.
Error Backup failed!
Error The application will close in 5 seconds!

**Comment by christianspecht** [comment]
`NLog.config` is the configuration file for SCM Backup's [logger](http://docs.scm-backup.org/en/latest/output-logging/), which should be in SCM Backup's main directory. This should be the case when you unzipped the download as-is.

I just tried to run SCM Backup via scheduled task on my machine (Windows 8.1).
When I just enter the path to the batch file in the "Edit Action" screen, I get errors as well (different message, just one line, disappears so quickly that I can't read it).

To get it to work, I had to fill "Start in (optional)" as well:
![Edit Action](https://user-images.githubusercontent.com/263372/53449073-5370b700-3a19-11e9-839c-f6012b32f03a.png)

Does that work for you?
